### PR TITLE
Notification changes

### LINKED ISSUE #54: Notify-send spam
On platforms without notify-send (e.g. macOS), "sh: notify-send: command not found" gets spammed within Discurses.

### LINKED ISSUE #63: Disabling notifications?
Just got Discurses up and running, I like it. but goodness gracious the notification spam is insane if you're in many servers. Is there any way to disabled the notifications entirely?

### PR DESCRIPTION
**Pull request by snus-kin**
Fixes #54 and Fixes #63 
- Adds a (hopefully) MacOS compatible notification
- Moves away from `os.system()` calls
- Adds config option to disable notifications

Somebody will need to test this on MacOS, as I do not have a working VM.

(I also hope I got all the `about.py` and `README.rst` changes right this time)

### CODE REVIEW

File: src/discurses/config.py [hunk 1]
```diff
@@ -35,23 +53,27 @@ def create_dir(directory):
...
-            "notify-send -i {avatar} \"{author} in chat with {users}:\" {content}"
-            .format(
-                avatar=shlex.quote(avatar),
-                author=shlex.quote(nickname),
-                content=shlex.quote(message.clean_content),
-                users=shlex.quote(', '.join(a.display_name for a in message.channel.recipients))))
+        if PLATFORM == "Linux":
+            linux_notify("-i {avatar} \"{author} in chat with {users}:\" {content}"
+                         .format(
+                             avatar=shlex.quote(avatar),
```
Comment (Ge0) [inline src/discurses/config.py]: I would `shlex.quote` inside the notification functions instead.

**Review by Ge0 — CHANGES_REQUESTED** [review]

File: src/discurses/config.py [hunk 2]
```diff
@@ -23,9 +27,23 @@ def create_dir(directory):
...
-
 create_dir(CACHE_AVATARS_PATH)
 
+def macos_notify(title, text):
+    """
+    Create a MacOS notification
+    Snippet from:
+    https://stackoverflow.com/questions/17651017/python-post-osx-notification/41318195#41318195
+    """
+    os.system("""
```
[the remark this answers is not in the document]
Comment (snus-kin) [reply src/discurses/config.py]: Silly mistake :)

File: src/discurses/config.py (hunk 1 above)
Comment (snus-kin) [reply src/discurses/config.py]: Hopefully fixed, I was worried about switching based on private message I guess.

**Comment by Ge0** [comment]
@topisani Do you think you could test the changes?

**Comment by topisani** [comment]
It looks good! Commencing merge...

**Pull request merged**